styles.css

```diff
@@ -1,4 +1,65 @@
 /*Maak hier jouw CSS!*/
 #container {
+    background-color: #3B0D11;
+    padding: 8px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 50px 190px 100px auto 50px;
+    grid-template-areas:
+            "a c c"
+            "b c c"
+            "b e d"
+            "b e none"
+            "f f f";
+}
+
+p {
+    margin: 8px;
+}
+
+#header {
+    background-color: #6A3937;
+    grid-area: a;
+}
+
+#main {
+    background-color: #706563;
+    color: white;
+    grid-area: b;
+}
+
+#big-aside {
+    background-color: #748386;
+    grid-area: c;
+}
+
+#small-aside {
+    background-color: #9DC7C8;
+    grid-area: d;
+}
+
+#paragraph {
+    background-color: #CADBC0;
+    grid-area: e;
+}
+
+#footer {
+    background-color: #F1F7ED;
+    grid-area: f;
+}
+
+@media only screen and (max-width: 576px) {
```

Comment:
In de uitwerkingen hebben ze hem mobile first gebouwd. ik zie dat jij hem andersom hebt gemaakt.